kratos/python/add_utilities_to_python.cpp

```diff
@@ -420,6 +421,14 @@ void AddUtilitiesToPython(pybind11::module& m)
     .def("TestGetExactIntegration",&ExactMortarIntegrationUtility<3,4>::TestGetExactIntegration)
     .def("TestGetExactAreaIntegration",&ExactMortarIntegrationUtility<3,4>::TestGetExactAreaIntegration)
     ;
+
+    // Sparse matrix multiplication utility
+    class_<SparseMatrixMultiplicationUtility, typename SparseMatrixMultiplicationUtility::Pointer>(m, "SparseMatrixMultiplicationUtility")
+    .def(init<>())
+    .def("MatrixMultiplicationSaad",&SparseMatrixMultiplicationUtility::MatrixMultiplicationSaad<CompressedMatrix, CompressedMatrix, CompressedMatrix>)
```

Comment:
what do you think of changing names as:

      MatrixMultiplicationSaad --> SpMM (this would be the default)
      MatrixMultiplicationRMerge -> SpMM_RMerge (another option)
      MatrixAdd -> SpAdd

at lease for SpMM we should suggest which one is the default (Saad's)

Reply:
Nope, you are wrong the selection is done in my app, one sec